test/powershell/Modules/Microsoft.PowerShell.Management/Get-ChildItem.Tests.ps1

```diff
@@ -4,12 +4,18 @@ Describe "Get-ChildItem" -Tags "CI" {
 
         BeforeAll {
             # Create Test data
-            $null = New-Item -Path $TestDrive -Name "a" -ItemType "File" -Force
-            $null = New-Item -Path $TestDrive -Name "B" -ItemType "File" -Force
-            $null = New-Item -Path $TestDrive -Name "c" -ItemType "File" -Force
-            $null = New-Item -Path $TestDrive -Name "D" -ItemType "File" -Force
-            $null = New-Item -Path $TestDrive -Name "E" -ItemType "Directory" -Force
-            $null = New-Item -Path $TestDrive -Name ".F" -ItemType "File" -Force | %{$_.Attributes = "hidden"}
+            $item_a = "a3fe710a-31af-4834-bc29-d0b584589838"
```

Comment:
The original filenames were `a`, `B`, `c`, `D`, `E`, and `.F`, in that letter casing, which appeared to have been a deliberate choice. To avoid disrupting any existing tests, I elected to keep the same letter casing in the GUID filenames.

It does look like I made a copy/paste error when I was generating GUID-based filenames. The filename for `$Item_E` should have begun with the letter/hex digit `E`. It now does.